pages/docs.html

```diff
@@ -108,76 +120,73 @@ <h3 id="alert">Alert</h3>
-          src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+        <p class="alert-message">This is a warning alert. You get a close icon as well</p><i class="far fa-times-circle alert-close fa-lg"></i>
       </div>
       <br>
 
       <!-- SIMPLE ALERT -->
       <div class="alert alert-simple">
-        <p class="alert-message">This is a simple alert. You get a close icon as well</p><img
-          src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+        <p class="alert-message">This is a simple alert. You get a close icon as well</p><i class="far fa-times-circle alert-close fa-lg"></i>
       </div>
 
       <br>
-     
+
       <!-- CODE SNIPPETS TO ADD RESPECTIVE ALERTS IN A PROJECT -->
       <section class="code-container">
         <pre class="pre-code">
           <code class="language-html">
   &lt;div class="alert alert-primary">
-    &lt;p class="alert-message">This is a primary alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a primary alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
   &lt;br>
   &lt;div class="alert alert-success">
-    &lt;p class="alert-message">This is a success alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a success alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
   &lt;br>
   &lt;div class="alert alert-danger">
-    &lt;p class="alert-message">This is a danger alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a danger alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
   &lt;br>
   &lt;div class="alert alert-warning">
-    &lt;p class="alert-message">This is a warning alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a warning alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
   &lt;br>
   &lt;div class="alert alert-simple">
-    &lt;p class="alert-message">This is a simple alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a simple alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
```

Comment:
Here also the Code indentation looks broken.

Reply:
the code snippet frame is also getting indented if i indent the html file. and as a result the in the output the code snippet is having empty space on the left which makes it look weird. tried carbon.now.sh but it ain't responsive

Reply:
> Here also the Code indentation looks broken.
the code snippet frame is also getting indented if i indent the html file. and as a result the in the output the code snippet is having empty space on the left which makes it look weird. tried carbon.now.sh but it ain't responsive